kubectl-cloudflow/config/config.go

```diff
@@ -343,11 +343,11 @@ func labelHasPrefix(label string) bool {
 
 func validateLabel(name string, prefix string) error {
 	// See https://github.com/kubernetes/kubernetes/issues/71140, IsDNS1123Subdomain and IsDNS1123Label do not allow uppercase letters.
-	labelPattern := regexp.MustCompile(`^[a-z0-9]{1}[a-z0-9\.\_\-]{0,61}[a-z0-9]{1}$`)
+	labelPattern := regexp.MustCompile(`^[a-z0-9A-Z]{1}[a-z0-9\.\_\-]{0,61}[a-z0-9A-Z]{1}$`)
```

Comment:
Have k8s changed its stance on uppercase in names?

Reply:
I have tried to add a label to a deployment with uppercase and it worked, so we should not prevent it.

Reply:
(the docs say it is legal as well)

Reply:
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set @olofwalker